### DISCUSSION TITLE
Pimcore admin issue after restore

### DISCUSSION THREAD

**Discussion by hellosmriti** [pimcore/pimcore#8513] [Q&A / Support]
Hi team,

Trying back-up and restore of a linux setup on windows. Database connection and login is successful. However after login, below screen appears:

![image](https://user-images.githubusercontent.com/54179597/111866112-280aed00-8991-11eb-850e-a2ced3033377.png)

Tried resolving by running _cache:clear_, _assets:insall web_ etc but nothing seems to help. Any help is appreciated.

Thanks in advance.

**Comment by jeykodev** [comment]
Hi @hellosmriti,

did you check the file permissions? 

https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Permissions.html

**Comment by hellosmriti** [comment]
@jeykodev I have tried doing this one in Windows and user is having Full Control over the folder.

**Comment by hellosmriti** [comment]
As suggested by somebody, enabling **disable-content-security** resolved this. However this patching works only in Chrome as of now.
